CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [0.1.1] - 02-Oct-2020
+
+**Milestone**: Hippopotamus(0.10.0.3)
+
+ Package  | Version  | Link
+---|---|---
+Symbol Bootstrap | v0.1.0 | [symbol-bootstrap](https://www.npmjs.com/package/symbol-bootstrap)
+
+- **New Command:** `symbol-bootstrap link` links the nodes' VRF and Voting keys to an existing network. This simplifies the node registration process to running networks like `testnet`.
+- **New Command:** `symbol-bootstrap report` generates rst and csv files from the configured server properties for documentation. Added `--report` flag to `config` and `start` commands.
```

Comment:
I am still not sure about the report command, we're still discussing in the other PR...

Reply:
I understood your concerns and the description.yml could be improved in the future. But I'm happy now with how the report looks. Let's see how it goes. We can evolve from there.  Thanks!